media-source/SourceBuffer-timestampOffset_getting.html

```diff
@@ -0,0 +1,50 @@
+<!doctype html>
+<html>
+<head>
+  <meta charset='utf-8'>
+  <title>Getting SourceBuffer#timestampOffset attribute</title>
+  <script src="/resources/testharness.js"></script>
+  <script src="/resources/testharnessreport.js"></script>
+</head>
+<body>
+<div id="log"></div>
+<script>
+var video = document.createElement('video');
+var mimes = ['video/webm; codecs="vorbis,vp8"', 'video/mp4'];
+
+//check the browser supports the MIME used in this test
+function isTypeSupported(mime) {
+    if(!MediaSource.isTypeSupported(mime)) {
+        this.step(function() {
+            assert_unreached("Browser doesn't support the MIME used in this test: " + mime);
+        });
+        this.done();
+        return false;
+    }
+    return true;
+}
+function testMedia(mime) {
+    async_test(function(t) {
+        if(!isTypeSupported.bind(t)(mime)) {
+            return;
+        }
+        var mediaSource = new MediaSource();
+        mediaSource.addEventListener('sourceopen', function(e) {
+            var sourceBuffer = mediaSource.addSourceBuffer(mime);
+            t.step(function() {
+                assert_equals(sourceBuffer.timestampOffset, 0);
```

Comment:
I'll add this coverage to mediasource-timestamp-offset in a separate CL that conforms to current MSE test infra.